site/pages/signer/authentication/social-login.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Oauth Authentication
+description: Authenticate a user using Oauth
+---
+
+# Social Login Authentication
+
+Email magic link authentication allows you to login and signup users using an Oauth provider, such as Google Sign-In or Facebook Login. You can also configure custom providers through Auth0.
+
+:::tip
+For setting up an account config, see the [Signer Quickstart](/signer/quickstart).
+:::
+
+## Authenticate a user
+
+:::code-group
+
+```ts twoslash [example.ts]
+import { signer } from "./signer";
+
+await signer.authenticate({
+  type: "oauth",
+  authProviderId: "google", // Choose between the auth providers you selected to support from your auth policy
+  mode: "redirect", // Alternatively, you can choose "popup" mode
```

Comment:
if `mode: "popup"` this requires you to set a prop on the `Signer` to `preparePopupOauth` right? 

we might want to add both code examples to this doc

Reply:
I think this is not needed if you set `enablePopupOauth` in the config, right?